lightningd/payalgo.c

```diff
@@ -325,7 +353,9 @@ static void json_pay(struct command *cmd,
 	}
 	pay->maxfeepercent = maxfeepercent;
 
-	pay->tries = 0;
+	pay->getroute_tries = 0;
+	pay->sendpay_tries = 0;
+	pay->fuzz = 0.75;
```

Comment:
We hand `pay->fuzz` into  towire_gossip_getroute_request() which takes a u32, so this turns into 0 fuzz.  That's probably not what we want; should we use the same default value as getroute(), or something else?